pigar/parser.py

```diff
@@ -61,7 +73,16 @@ def _read_code(fpath):
         code = ""
         for cell in nb.cells:
             if cell.cell_type == "code":
-                code += cell.source + "\n"
+                if _transformer_manager:
+                    code += _transformer_manager.transform_cell(cell.source) \
+                        + "\n"
```

Comment:
I think the problem is the `+ "\n"`, could you remove it and run the test to see what happens?

Ref:
https://github.com/ipython/ipython/blob/86d24741188b0cedd78ab080d498e775ed0e5272/IPython/core/inputtransformer2.py#L588-L597

Reply:
done